### PR TITLE
Fixes #430, fixes #729: (Offset for loading dialog for the login screen)

### LINKED ISSUE #430: More minimalistic loading screen
From this: 
image
To this: 
image

### LINKED ISSUE #729: Progress Bar not properly placed
screenshot_2018-02-14-00-51-20-873_openfoodfacts github scrachx openfood

Progress bar is getting trimmed at the top in some of the devices.
I want to work on this.

### PR DESCRIPTION
**Pull request by ManognaMutyala**
screenshot_20180214-160356
Places the loading message properly.Screen would be something like this

### CODE REVIEW

**Comment by teolemon** [comment]
What about a snackbar ?
image

**Comment by ManognaMutyala** [comment]
So do you want me to add snackbar with welcome information @teolemon ?

**Comment by teolemon** [comment]
It was the format. A snackbar with "Loading" ? and then one with "Connected" ?

**Comment by ManognaMutyala** [comment]
No No it is not snackbar !!! A separate library known as loadToast is used. More information about it-https://github.com/code-mc/loadtoast

**Comment by ross-holloway94** [comment]
@ManognaMutyala I believe what @teolemon is trying to say is: is it possible to put "loading..." and "connected" into a snackbar instead.

**Comment by policorn** [comment]
Hi @ManognaMutyala, are you still working on this?

**Comment by ManognaMutyala** [comment]
Hi @policorn  and yeah I am still working on this!!

**Comment by ManognaMutyala** [comment]
I am working on it @teolemon  @ross-holloway94

**Comment by ManognaMutyala** [comment]
I just wanted to make things clear @ross-holloway94 @teolemon  !! Do you want me to completely replace loadToast and use Snackbar in its place?

**Comment by Karljoones** [comment]
@ManognaMutyala Yes, they would like you to replace it with a snackbar :) How is this coming along?

**Comment by ross-holloway94** [comment]
@ManognaMutyala That would look great, thanks.

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/views/LoginActivity.java [hunk 1]
```diff
@@ -104,20 +104,18 @@ protected void attemptLogin() {
...
         }
 
-        final LoadToast lt = new LoadToast(this);
-        save.setClickable(false);
-        lt.setText(getString(R.string.toast_retrieving));
-        lt.setBackgroundColor(ContextCompat.getColor(this,R.color.blue));
-        lt.setTextColor(ContextCompat.getColor(this,R.color.white));
-        lt.show();
+        Snackbar snackbar = Snackbar
+                .make(linearLayout, "Loading", Snackbar.LENGTH_LONG);
```
Comment (Karljoones) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/views/LoginActivity.java]: Can we make the snackbar text come from strings.xml? :)

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/views/LoginActivity.java [hunk 2]
```diff
@@ -152,7 +150,11 @@ public void onResponse(Call<ResponseBody> call, Response<ResponseBody> response)
                             break;
                         }
                     }
-                    lt.success();
+                     Snackbar snackbar = Snackbar
+                            .make(linearLayout, "Connected", Snackbar.LENGTH_LONG);
```
Comment (Karljoones) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/views/LoginActivity.java]: And here too? :)

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/views/LoginActivity.java (hunk 1 above)
Comment (ManognaMutyala) [reply app/src/main/java/openfoodfacts/github/scrachx/openfood/views/LoginActivity.java]: Yeah sure !! I will change it @Karljoones

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/views/LoginActivity.java (hunk 2 above)
Comment (ManognaMutyala) [reply app/src/main/java/openfoodfacts/github/scrachx/openfood/views/LoginActivity.java]: Yeah sure!!

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/views/LoginActivity.java (hunk 1 above)
Comment (Karljoones) [reply app/src/main/java/openfoodfacts/github/scrachx/openfood/views/LoginActivity.java]: Thanks!

**Comment by Karljoones** [comment]
How is this coming along @ManognaMutyala?

**Comment by Karljoones** [comment]
Hi @ManognaMutyala, are the changed implemented and is this ready to be reviewed?

**Comment by ManognaMutyala** [comment]
I just now resolved the conflicts @Karljoones, Sorry for the delay!

**Comment by Karljoones** [comment]
No problem at all @ManognaMutyala, thank you for making the changes :)

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -92,6 +92,9 @@
     </string-array>
 
     <!-- Content Description -->
+    <string name="connection">Connected</string>
+    <string name="fastscroller_track">FastScroller track</string>
```
Comment (teolemon) [inline app/src/main/res/values/strings.xml]: I don't see why those strings are here

Comment (policorn) [reply app/src/main/res/values/strings.xml]: I think those result from a merge from OFF upstream to his development branch

**Comment by Karljoones** [comment]
What's happening with this PR @teolemon @policorn?

**Comment by policorn** [comment]
Hi @ManognaMutyala can you fix the conflict in LoginActivity.java so we can merge this one?

**Comment by ManognaMutyala** [comment]
Hi @policorn  I made changes so that it can be merged!!!

**Comment by Karljoones** [comment]
@ManognaMutyala please have a look at the travis build to see if you can resolve the reason why it is failing!

**Comment by ManognaMutyala** [comment]
Travis build didn't run at all.**This job was cancelled because the "Auto Cancellation" feature is currently enabled, and a more recent build (#5381) for pull request #739 came in while this job was waiting to be processed.** this was the message provided

**Comment by teolemon** [comment]
Currently upgrading my hard drive. Can you add a screenshot so that I can review the UX and merge from mobile ?

**Comment by ManognaMutyala** [comment]
screenshot_20180218-224230

**Comment by policorn** [comment]
Two tests have failed due to a connection error

```
openfoodfacts.github.scrachx.openfood.network.OpenFoodAPIServiceTest > getProductByAdditive_e301_productsFound FAILED
    java.net.SocketTimeoutException at OpenFoodAPIServiceTest.java:270
        Caused by: java.net.SocketException at OpenFoodAPIServiceTest.java:270
openfoodfacts.github.scrachx.openfood.network.OpenFoodAPIServiceTest > byCountry FAILED
    java.net.SocketTimeoutException at OpenFoodAPIServiceTest.java:154
        Caused by: java.net.SocketException at OpenFoodAPIServiceTest.java:154
```

I've restarted the CI job to check if it was caused by a temporally connection error

**Comment by policorn** [comment]
Fixed a conflict in merge; @ManognaMutyala please verify if you're ok with it

**Comment by Karljoones** [comment]
@policorn test completed with no issues, good to merge?

**Comment by policorn** [comment]
Thanks @ManognaMutyala, @Karljoones! Merged!

**Pull request merged**